rust/operator/src/lib.rs

```diff
@@ -519,45 +528,41 @@ impl HdfsState {
             });
         }
 
+        // TODO: if we create this it will be under root and we cannot access via stackable user
+        //   HDFS will create it itself under stackable user
+        // One mount for data_dir
+        // if let Some(dir) = data_dir {
+        //     cb.add_volume_mount("data", dir);
+        //     pod_builder.add_volume(
+        //         VolumeBuilder::new("data")
+        //             .with_empty_dir(Some(""), None)
+        //             .build(),
+        //     );
+        // }
+
```

Comment:
In the Druid operator we don't use emptyDir but hostPath which survives pods and we run the service as root in that case.